drivers/SmartThings/matter-sensor/src/air-quality-sensor/init.lua

```diff
@@ -127,9 +157,14 @@ local function configure(driver, device)
   local pm10_eps = device:get_endpoints(clusters.Pm10ConcentrationMeasurement.ID)
   local radon_eps = device:get_endpoints(clusters.RadonConcentrationMeasurement.ID)
   local tvoc_eps = device:get_endpoints(clusters.TotalVolatileOrganicCompoundsConcentrationMeasurement.ID)
+  local tvoc_eps_level = device:get_endpoints(clusters.TotalVolatileOrganicCompoundsConcentrationMeasurement.ID, {feature_bitmap = clusters.TotalVolatileOrganicCompoundsConcentrationMeasurement.types.Feature.LEVEL_INDICATION})
 
   -- check to see if device can switch to a profile with less capabilities to support
-  if #CO_eps > 0 or #NO2_eps > 0 or #ozone_eps > 0 or #formaldehyde_eps > 0 or #pm1_eps > 0 or
+  if #CO_eps > 0 and #CO2_eps > 0 and #NO2_eps > 0 and #ozone_eps > 0 and #formaldehyde_eps > 0 and #pm1_eps > 0 and
```

Comment:
It'd be good to add a test case for this.

Reply:
Added in 2f6d6be9343a806c29c8f9bfaa5122623aa0d116.